plugin/build.gradle.kts

```diff
@@ -9,11 +9,7 @@ plugins {
 }
 
 dependencies {
-    // Using compileOnly because every android project applies
-    // its own version of the Android Plugin.
-    // We're staying on 3.0.1 so we don't accidentally use APIs
-    // that are not available in the client project
-    compileOnly("com.android.tools.build:gradle:3.0.1")
+    compileOnly("com.android.tools.build:gradle:3.3.0-alpha11")
```

Comment:
`3.3.0-alpha12` was released yesterday. Or merge as-is and bump before release?

Reply:
I'll bump now